.github/workflows/main.yml

```diff
@@ -104,8 +108,14 @@ jobs:
       - name: wmass combine setup
         run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/combine/setupCombineWMass.py -i $OUTFILE --verbose 4 --lumiScale $LUMI_SCALE
 
+      - name: dilepton combine setup
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/combine/setupCombineWMass.py -i $OUTFILE_DILEPTON --fitvar ptll-yll --verbose 4 --lumiScale $LUMI_SCALE
+
+      - name: aggregate combine inputs
+        run: cp -p ZMassDilepton_ptll_yll/* WMass_pt_eta/
+
       - name: wmass combine fit 
-        run: cmssw-cc7 --command-to-run scripts/ci/run_combine.sh /home/c/cmsmwbot/combinetf/CMSSW_10_6_30/src/ mass WMass_pt_eta WMass_plus.txt WMass_minus.txt 
+        run: cmssw-cc7 --command-to-run scripts/ci/run_combine.sh /home/c/cmsmwbot/combinetf/CMSSW_10_6_30/src/ mass WMass_pt_eta WMass_plus.txt WMass_minus.txt ZMassDilepton_minus.txt ZMassDilepton_plus.txt
```

Comment:
Can you check if this works with the full paths?

Reply:
It will work with full absolute paths, but not with relative paths which would be needed here, since the working directory is already WMass_pt_eta.

So at least this would require some changes to the run_combine.sh script to make the paths more flexible